frontend/packages/git-service/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@console/git-service",
+  "version": "0.0.0-fixed",
+  "description": "Git Service Utility for Git Validation and Language Detection.",
+  "private": true,
+  "main": "src/index.ts",
+  "license": "MIT",
+  "dependencies": {
```

Comment:
lots of new dependencies
What's the overall size addition due to these?
cc @spadgett

Reply:
I know this adds a lot of dependency since its not making the API requests itself but delegating it to specific git clients. We could write our own implementation of just the API requests that we need and not use any of the external clients to reduce the dependency but not sure if that is a viable option at this point of time considering the 4.3 timelines. We could look at it as a tech debt later on.

Reply:
Just curious at the bundle size analyzer result of the new additions. Make sure at aren't surprised.

We will be lazy loading the libs on specific pages anyways.